dnf-plugins-extras.spec

```diff
@@ -102,8 +104,8 @@ BuildRequires:  python3-systemd
 %{?systemd_requires}
 
 %description -n python3-dnf-plugin-system-upgrade
-System Upgrade Plugin for DNF, Python 3 version. Enables offline system upgrades
-using the "dnf system-upgrade" command.
+System Upgrade Plugin for DNF, Python 3 version. Enables offline system upgrades, distrosync
```

Comment:
Probably `upgrades *and* distrosync` instead of the comma

Reply:
Good suggestion, thanks.